lens/main.scrbl

```diff
@@ -4,10 +4,10 @@
 
 @defmodule[lens]
 
-This library includes functions and forms for working with @italic{lenses}.
-A lens is a value that operates on some small piece of a larger
-structure. Think of them as a more general representation of getters and
-setters in object-oriented languages.
+This library includes functions and forms for working with
+@deftech[#:tag 'lens]{lenses}. A lens is a value that operates on some
```

Comment:
*sigh* Yes, yes it is. I'm not sure how I was stupid enough to both push this without testing that change *and* to screw it up so badly. Anyway, I fixed it now.